src/res/translation/translation_nl_NL.ts

```diff
@@ -239,7 +239,7 @@
         <location filename="../../audiomixerboard.cpp" line="248"/>
         <location filename="../../audiomixerboard.cpp" line="264"/>
         <source>Pan</source>
-        <translation>Balans</translation>
+        <translation>Bal.</translation>
```

Comment:
I vaguely remember that this was changed the other way round recently. Maybe some agreement is needed to keep it one way. :)

Reply:
Correct, it was changed the other way around recently, but because of the length of the text it is causing shifting the screen when the muted speaker icon is displayed in compact skin mode. Therefore going back to the abbreviated text.